### DISCUSSION TITLE
Welcome to arabic-tutorial-book Discussions!

### DISCUSSION THREAD

**Discussion by adamiturabi** [adamiturabi/arabic-tutorial-book#123] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by taominator** [comment]
Assalamu 'alaikum brother

This is an amazing intiative. I am impressed by the intuitiveness of the book especially for a beginner. May Allah bring khair through your work. I was sad when I find out it's not complete. Would you please be able to provide simimlar resources like this?

**Comment by adamiturabi** [comment]
wa alaykum assalaam Brother,
Thank you for your comment and your dua. Aameen.
I'm continuing to work on the book (slowly but steadily). InshaaAllah I hope it will get completed.
There are some reference works that I have cited here: https://adamiturabi.github.io/arabic-tutorial-book/content/refs.html
Wright's and Fischer's grammar books are good, but they are reference works.

**Comment by 0xma1ware** [comment]
السلام عليكم ورحمة الله وبركاته

This is truly an amazing effort Mashallah. I stumbled upon this repository while considering digitizing and translating my own notes on Arabic Grammar into English for the ease of the beginner student. 

I'm absolutely blown away by the amount of effort you have put into this repository. I will be more than happy to review this and perhaps later on even contribute if the schedule permits :) 

جزاك الله خيرًا ونفع الله بك الأمة

**Reply by adamiturabi** [reply]
وعليكم السلام ورحمة الله وبركاته

آمين. تقبل الله منا ومنكم.

Thank you for your kind words. And thank you very much for your offer to review and contribute.
جزاك الله خيرا

**Reply by adamiturabi** [reply]
Beware , though, that many of the (especially initial) chapters need massive editing and/or re-writes.  Just so that you don't review material that is intended to change anyway, إن شاء الله.
